conifer/converters/__init__.py

```diff
@@ -22,6 +24,74 @@ def get_converter(converter):
 def get_available_converters():
   return [k for k in _converter_map.keys()]
 
+
+def compute_float_lsb(config):
+    """Given a fixed point precision, compute the least significant bit.
+    If the precision is float, return None and let the other method compute
+    the smallest delta for each float.
+
+    Args:
+        config (Dict): backend configuration
+
+    Returns:
+        float|None: offset
+    """
+    if config is None or "Precision" not in config or config["Precision"] == "float":
+        # In floating point, the smallest difference depends on the magnitude of the number
+        # Use the nextafter numpy method
+        return None
+    ap_fixed = config["Precision"]
```

Comment:
We should be careful to select the `ThresholdPrecision`. That would be easy using the `MultiPrecisionConfig` from `conifer.backends.common` but that might get messy with imports, not sure.
https://github.com/thesps/conifer/blob/master/conifer/backends/common.py#L64